### PR TITLE
Fixed hack/verify-golint.sh reported errors 

### LINKED ISSUE #63065: fixes on errors reported by `make verify`
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug
> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind feature

**What happened**:
`make verify` reports following errors-

**What you expected to happen**:
This is the required fix reported by PR #62895 processing.

**How to reproduce it (as minimally and precisely as possible)**:
run `make verify` from k8s source.

**Anything else we need to know?**:
Detailed list of ERRORS reported by `make verify`
```
========================
FAILED TESTS
========================
hack/make-rules/../../hack/verify-gofmt.sh
hack/make-rules/../../hack/verify-golint.sh
hack/make-rules/../../hack/verify-govet.sh
make: *** [verify] Error 1
=======================
```
```
Verifying verify-gofmt.sh

+++ Running case: verify.gofmt
+++ working dir: /Users/mayank.gaikwad/work/go-workspace/src/k8s.io/kubernetes
+++ command: bash "hack/make-rules/../../hack/verify-gofmt.sh"
diff -u ./staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go.orig ./staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go
--- ./staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go.orig	2018-04-24 11:34:26.000000000 +0530
+++ ./staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go	2018-04-24 11:34:26.000000000 +0530
@@ -44,10 +44,10 @@

 	publicNamer := &ExceptionNamer{
 		Exceptions: map[string]string{
-			// these exceptions are used to deconflict the generated code
-			// you can put your fully qualified package like
-			// to generate a name that doesn't conflict with your group.
-			// "k8s.io/apis/events/v1beta1.Event": "EventResource"
+		// these exceptions are used to deconflict the generated code
+		// you can put your fully qualified package like
+		// to generate a name that doesn't conflict with your group.
+		// "k8s.io/apis/events/v1beta1.Event": "EventResource"
 		},
 		KeyFunc: func(t *types.Type) string {
 			return t.Name.Package + "." + t.Name.Name
@@ -56,10 +56,10 @@
 	}
 	privateNamer := &ExceptionNamer{
 		Exceptions: map[string]string{
-			// these exceptions are used to deconflict the generated code
-			// you can put your fully qualified package like
-			// to generate a name that doesn't conflict with your group.
-			// "k8s.io/apis/events/v1beta1.Event": "eventResource"
+		// these exceptions are used to deconflict the generated code
+		// you can put your fully qualified package like
+		// to generate a name that doesn't conflict with your group.
+		// "k8s.io/apis/events/v1beta1.Event": "eventResource"
 		},
 		KeyFunc: func(t *types.Type) string {
 			return t.Name.Package + "." + t.Name.Name
@@ -68,10 +68,10 @@
 	}
 	publicPluralNamer := &ExceptionNamer{
 		Exceptions: map[string]string{
-			// these exceptions are used to deconflict the generated code
-			// you can put your fully qualified package like
-			// to generate a name that doesn't conflict with your group.
-			// "k8s.io/apis/events/v1beta1.Event": "EventResource"
+		// these exceptions are used to deconflict the generated code
+		// you can put your fully qualified package like
+		// to generate a name that doesn't conflict with your group.
+		// "k8s.io/apis/events/v1beta1.Event": "EventResource"
 		},
 		KeyFunc: func(t *types.Type) string {
 			return t.Name.Package + "." + t.Name.Name
diff -u ./pkg/security/podsecuritypolicy/provider_test.go.orig ./pkg/security/podsecuritypolicy/provider_test.go
--- ./pkg/security/podsecuritypolicy/provider_test.go.orig	2018-04-24 11:34:32.000000000 +0530
+++ ./pkg/security/podsecuritypolicy/provider_test.go	2018-04-24 11:34:32.000000000 +0530
@@ -1017,7 +1017,7 @@
 		},
 		Spec: api.PodSpec{
 			SecurityContext: &api.PodSecurityContext{
-				// fill in for test cases
+			// fill in for test cases
 			},
 			Containers: []api.Container{
 				{
@@ -1041,7 +1041,7 @@
 		},
 		Spec: v1.PodSpec{
 			SecurityContext: &v1.PodSecurityContext{
-				// fill in for test cases
+			// fill in for test cases
 			},
 			Containers: []v1.Container{
 				{
diff -u ./pkg/controller/serviceaccount/tokens_controller_test.go.orig ./pkg/controller/serviceaccount/tokens_controller_test.go
--- ./pkg/controller/serviceaccount/tokens_controller_test.go.orig	2018-04-24 11:34:33.000000000 +0530
+++ ./pkg/controller/serviceaccount/tokens_controller_test.go	2018-04-24 11:34:33.000000000 +0530
@@ -474,7 +474,7 @@

 			AddedSecret:     serviceAccountTokenSecretWithNamespaceData([]byte("custom")),
 			ExpectedActions: []core.Action{
-				// no update is performed... the custom namespace is preserved
+			// no update is performed... the custom namespace is preserved
 			},
 		},

@@ -539,7 +539,7 @@

 			UpdatedSecret:   serviceAccountTokenSecretWithNamespaceData([]byte("custom")),
 			ExpectedActions: []core.Action{
-				// no update is performed... the custom namespace is preserved
+			// no update is performed... the custom namespace is preserved
 			},
 		},

diff -u ./pkg/kubectl/cmd/config/create_authinfo_test.go.orig ./pkg/kubectl/cmd/config/create_authinfo_test.go
--- ./pkg/kubectl/cmd/config/create_authinfo_test.go.orig	2018-04-24 11:34:35.000000000 +0530
+++ ./pkg/kubectl/cmd/config/create_authinfo_test.go	2018-04-24 11:34:35.000000000 +0530
@@ -142,7 +142,7 @@
 		},
 		{
 			flags: []string{
-				// No name for authinfo provided.
+			// No name for authinfo provided.
 			},
 			wantCompleteErr: true,
 		},

Run ./hack/update-gofmt.sh
+++ exit code: 1
+++ error: 1
FAILED   verify-gofmt.sh	16s
Verifying verify-golint.sh

+++ Running case: verify.golint
+++ working dir: /Users/mayank.gaikwad/work/go-workspace/src/k8s.io/kubernetes
+++ command: bash "hack/make-rules/../../hack/verify-golint.sh"
Errors from golint:
kubernetes/cluster/images/etcd-version-monitor/etcd-version-monitor.go:71:1: comment on exported type EtcdVersion should be of the form "EtcdVersion ..." (with optional leading article)
kubernetes/cluster/images/etcd-version-monitor/etcd-version-monitor.go:135:1: comment on exported type GRPCRequestLabels should be of the form "GRPCRequestLabels ..." (with optional leading article)
kubernetes/examples/guestbook-go/main.go:35:1: exported function ListRangeHandler should have comment or be unexported
kubernetes/examples/guestbook-go/main.go:43:1: exported function ListPushHandler should have comment or be unexported
kubernetes/examples/guestbook-go/main.go:51:1: exported function InfoHandler should have comment or be unexported
kubernetes/examples/guestbook-go/main.go:56:1: exported function EnvHandler should have comment or be unexported
kubernetes/examples/guestbook-go/main.go:69:1: exported function HandleError should have comment or be unexported
pkg/kubelet/certificate/bootstrap/bootstrap.go:126:2: redundant if ...; err != nil check, just return error instead.

Please review the above warnings. You can test via "golint" and commit the result.
If the above warnings do not make sense, you can exempt this package from golint
checking by adding it to hack/.golint_failures (if your reviewer is okay with it).

+++ exit code: 1
+++ error: 1
FAILED   verify-golint.sh	190s
Verifying verify-govet.sh

+++ Running case: verify.govet
+++ working dir: /Users/mayank.gaikwad/work/go-workspace/src/k8s.io/kubernetes
+++ command: bash "hack/make-rules/../../hack/verify-govet.sh"
can't load package: package k8s.io/kubernetes/kubernetes/examples: code in directory /Users/mayank.gaikwad/work/go-workspace/src/k8s.io/kubernetes/kubernetes/examples expects import "k8s.io/kubernetes/examples"
make[1]: *** [vet] Error 1
+++ exit code: 2
+++ error: 1
FAILED   verify-govet.sh	299s
```

**Environment**:
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.1", GitCommit:"d4ab47518836c750f9949b9e0d387f20fb92260b", GitTreeState:"clean", BuildDate:"2018-04-13T22:27:55Z", GoVersion:"go1.9.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-04-10T12:46:31Z", GoVersion:"go1.9.4", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration:

- OS (e.g. from /etc/os-release):
``` macOS 10.13.4```

- Kernel (e.g. `uname -a`):
```Darwin mayankg.local 17.5.0 Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018; root:xnu-4570.51.1~1/RELEASE_X86_64 x86_64 i386 MacBookPro11,4 Darwin```

- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by mgdevstack**
**What this PR does / why we need it**:
Fixes errors reported by `hack/verify-golint.sh` while running `make verify`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #63065 

**Special notes for your reviewer**:
Fixing `hack/verify-gofmt.sh` reported error for `k8s.io/code-generator/cmd/client-gen/generators/client_generator.go` mentioned in #63065 throws error. So didn't modify that file.

**Release note**:

```release-note
NONE
```
/sig-testing-bugs

### CODE REVIEW

**Comment by mgdevstack** [comment]
cc @BenTheElder

File: examples/guestbook-go/main.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ var (
 	slavePool  *simpleredis.ConnectionPool
 )
 
+// ListRangeHandler :
```
Comment (cblecker) [inline examples/guestbook-go/main.go]: Please drop changes to `examples/`.

File: hack/.golint_failures [hunk 2]
```diff
@@ -9,7 +9,6 @@ cmd/kubeadm/app/apis/kubeadm/v1alpha1
 cmd/kubelet/app
 cmd/kubelet/app/options
 cmd/kubemark
-examples/guestbook-go
```
Comment (cblecker) [inline hack/.golint_failures]: Please remove this change.

Comment (mgdevstack) [reply hack/.golint_failures]: Removing change. Just an quick question,
1. Why golint reports error for these packages while running '`make verify`, even if they are added into `.golint_failures` list? 
2. Should we keep ignoring linting errors if entries found in `.golint_failures`?

File: examples/guestbook-go/main.go [hunk 3]
```diff
@@ -32,6 +32,7 @@ var (
...
 func ListRangeHandler(rw http.ResponseWriter, req *http.Request) {
```
[the remark this answers is not in the document]
Comment (mgdevstack) [reply examples/guestbook-go/main.go]: seems `examples/` has been removed in upstream. Removing golint changes.

File: hack/.golint_failures (hunk 2 above)
Comment (mgdevstack) [reply hack/.golint_failures]: @cblecker should we remove `examples/guestbook-go` from `.golint_failures`, as `examples/` has been removed from upstream ?

File: hack/.golint_failures [hunk 4]
```diff
@@ -9,6 +9,7 @@ cmd/kubeadm/app/apis/kubeadm/v1alpha1
 cmd/kubelet/app
 cmd/kubelet/app/options
 cmd/kubemark
+examples/guestbook-go
```
Comment (cblecker) [inline hack/.golint_failures]: this package no longer exists. Can you please remove this and squash into a single commit?

Comment (mgdevstack) [reply hack/.golint_failures]: Done @cblecker

**Pull request merged**